ts/screens/onboarding/OnboardingServicesPreferenceScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { SafeAreaView, ScrollView } from "react-native";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../utils/emptyContextualHelp";
+import ServicesContactComponent from "../profile/components/services/ServicesContactComponent";
+import { InfoBox } from "../../components/box/InfoBox";
+import { H5 } from "../../components/core/typography/H5";
+import { IOColors } from "../../components/core/variables/IOColors";
+import I18n from "../../i18n";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import { confirmButtonProps } from "../../features/bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import { GlobalState } from "../../store/reducers/types";
+import { Dispatch } from "../../store/actions/types";
+import { servicesOptinCompleted } from "../../store/actions/onboarding";
+import { IOStyles } from "../../components/core/variables/IOStyles";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+const OnboardingServicesPreferenceScreen = (
+  props: Props
+): React.ReactElement => (
+  <BaseScreenComponent contextualHelp={emptyContextualHelp}>
```

Comment:
Just as warning to keep in mind, without `headerTitle` we don't have the automatic handling of the screen reader focus.

Reply:
You're right, I guess we can add the same screen title as on preferences screen. Will check with design team